src/Backend/Modules/Faq/Actions/DeleteCategory.php

```diff
@@ -41,7 +41,7 @@ public function execute(): void
             return;
         }
 
-        $this->record = (array) BackendFaqModel::getCategory($this->id);
+        $this->record = BackendFaqModel::getCategory($this->id);
```

Comment:
Don't change anything in the actions, only in the model.

Reply:
this (array) conversion is obsolete, as it is enforced by php7 return type hinting

Reply:
ok